packages/@divvi/mobile/src/statsig/index.ts

```diff
@@ -1,19 +1,17 @@
-import * as _ from 'lodash'
+import { DynamicConfig, StatsigUser } from '@statsig/react-native-bindings'
+import _ from 'lodash'
 import { LaunchArguments } from 'react-native-launch-arguments'
-import { startOnboardingTimeSelector } from 'src/account/selectors'
 import { ExpectedLaunchArgs, STATSIG_ENABLED } from 'src/config'
 import { FeatureGates } from 'src/statsig/constants'
+import { getDefaultStatsigUser } from 'src/statsig/selector'
 import {
   StatsigDynamicConfigs,
   StatsigExperiments,
   StatsigFeatureGates,
   StatsigParameter,
 } from 'src/statsig/types'
 import Logger from 'src/utils/Logger'
-import { walletAddressSelector } from 'src/web3/selectors'
-import { EvaluationReason } from 'statsig-js'
-import { DynamicConfig, Statsig, StatsigUser } from 'statsig-react-native'
-
+import StatsigClientSingleton from './client'
```

Comment:
```suggestion
import StatsigClientSingleton from './client'

```
nit: blank line after imports